SafeGraph_py_functions/SafeGraph_py_functions.py

```diff
@@ -148,14 +156,14 @@ def explode_json_array(df_, array_column = 'visits_by_day', new_col='day_visit_c
 ### ---------------------------------------CORE, GEO, AND PATTERNS SECTION -----------------------------------------------
 
 
-def read_core_folder(path_to_core, *args, **kwargs):
+def read_core_folder(path_to_core, compression='gzip',*args, **kwargs):
     core_files = glob.glob(os.path.join(path_to_core, "*.csv.gz"))
     print(f"You are about to load in {len(core_files)} core files")
 
     li = []
     for core in core_files:
         print(core)
-        df = pd.read_csv(core, *args, **kwargs, compression='gzip',
+        df = pd.read_csv(core, *args, **kwargs, compression=compression,
```

Comment:
@Trippl7777 all standard arguments (e.g., compression) should come before *args and **kwargs.